source/_drafts/videojs-contrib-ads-6.md

```diff
@@ -0,0 +1,41 @@
+---
+title: videojs-contrib-ads 6
```

Comment:
Maybe "videojs-contrib-ads v6 Released"?

Reply:
I think "videojs-contrib-ads 6" is a bit more evergreen: this post will still be about what changed in contrib-ads 6 even after it's not "news"